spec/uploader_spec.rb

```diff
@@ -5,7 +5,14 @@
 include Txdb
 
 describe Uploader, test_db: true do
-  let(:database) { TestDb.database }
+  let(:database) do
+    TestDb.setup do
+      create_table(:foo) do
```

Comment:
Yes, this one is necessary. The uploader iterates over all the tables in the given database and calls the Transifex API. There has to be at least one table for this to work.